### PR TITLE
Add instructions to gen section

### PR DESCRIPTION
**Pull request by anthonygc**
# Generation instruction for README

## Description

Added some instructions for the commands used for the generation of yml files.

## Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @anthonygc 

## Reminder

 - All GitHub Actions should be in a passing state before any pull request is merged.
 - All PRs must be reviewed by at least one team member and one member of the Integration team!
 - Any issues this PR closes are tagged in the description!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ Output team
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to configure tests.
+
```
Comment (Yanqiao4396) [inline README.md:52]: Could you please try to simplify the messages I show below? They are what users need to know for calling the implementation function.
     Warning: User should provide precise names of folders or files and provide correct character case as input
     Warning: The empty folder or the folders only containing files starting with __ or . will be ignore
     Warning: Don't name folder or file starting with dot or double underscore. They will be ignored
     If you input a folder name, all the sub-dirs and files will be reserved except the ones starting with . or __
     Warning: Only folder names in root dir or the one after root will be checked. Don't put the target folders too deep
     The file names have no above problem. You can put it in any dir.

Comment (Yanqiao4396) [reply README.md:52]: Also, rn here is another input argument. User can provide relative directory to decide where get started to scan

**Review by Yanqiao4396 — CHANGES_REQUESTED** [review]
Please check the comment I leave

File: README.md [hunk 2]
```diff
@@ -48,7 +48,22 @@ Output team
...
+In the terminal within the main directory.
+
+Use command `gatorgrade generate <path> <root directory>` to gen a yml file.
+
+- User should provide precise names of folders or files.
+
+- The user should also provide correct character case as input.
+
+- All folder(s) containing files starting with `__` or `.` will be ignore
```
Comment (Michionlion) [inline README.md:61]:
```suggestion
- All folder(s) containing files starting with `__` or `.` will be ignored
```

File: README.md [hunk 3]
```diff
@@ -48,7 +48,22 @@ Output team
...
+
+- Don't name folder or file starting with dot or double underscore.
+- These files will be ignored.
```
Comment (Michionlion) [inline README.md:64]: This is a duplicate of the point above.

File: README.md [hunk 4]
```diff
@@ -48,7 +48,22 @@ Output team
...
+
+All sub-dirs and files will be reserved except ones starting with `.` or `__`
```
Comment (Michionlion) [inline README.md:66]: It's not clear what "reserved" means here - could you clarify it?

**Review by Michionlion — CHANGES_REQUESTED** [review]
This looks good! A few changes and fixes that I think could improve the documentation are below.

File: README.md (hunk 4 above)
Comment (anthonygc) [reply README.md:71]: Thanks!

File: README.md [hunk 5]
```diff
@@ -53,7 +53,21 @@ red, while 100% of checks passed will appear in green.
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to create assignments.
+
+In the terminal within the main directory.
+
+Use command gatorgrade generate <TARGET_PATH_LIST> to generate a yml file.
```
Comment (Michionlion) [inline README.md:60]: You'll probably still want the backticks ``` around this command.

**Review by Michionlion — DISMISSED** [review]
This looks good!

**Review by Nathandloria — DISMISSED** [review]
Looks great!

File: README.md [hunk 6]
```diff
@@ -53,7 +53,21 @@ red, while 100% of checks passed will appear in green.
...
+
+- User should provide precise name of the of target list .
+
+- The user should also provide correct character case as input.
+
+- All folder(s) containing files starting with `__` or `.` will be ignored
+
+- Don't name folder or file starting with dot or double underscore.
+
+All sub-dirs and files will be stored except ones starting with `.` or `__`
```
Comment (Michionlion) [inline README.md:70]: These three points are redundant.

File: README.md [hunk 7]
```diff
@@ -53,7 +53,21 @@ red, while 100% of checks passed will appear in green.
...
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to create assignments.
+
+In the terminal within the main directory.
+
+Use `gatorgrade generate <TARGET_PATH_LIST>` to generate `gatorgrade.yml` file.
+
+- User should provide precise name of the of target list .
```
Comment (Michionlion) [inline README.md:62]:
```suggestion
- User should provide full paths in the target list .
```

**Review by Michionlion — CHANGES_REQUESTED** [review]

**Review by Nathandloria — DISMISSED** [review]
Looks good! Keep in mind your conversation with Maria for details on how to add more content in the future.

**Review by Nathandloria — APPROVED** [review]

**Pull request merged**